common/src/test/java/com/datastrato/gravitino/config/TestConfigEntry.java

```diff
@@ -24,13 +25,60 @@ public void initializeConfigMap() {
     configMap.put("gravitino.test.string", "test-string");
     configMap.put("gravitino.test.string.alt1", "test-string1");
     configMap.put("gravitino.test.string.alt2", "test-string2");
+    configMap.put("gravitino.test.stringList", "test-string1,test-string2,test-string3");
+    configMap.put("gravitino.test.integerList", "1,2,3");
   }
 
   @AfterEach
   public void clearConfigMap() {
     configMap.clear();
   }
 
+  @Test
+  public void testStrConfValueList() {
+    ConfigEntry<List<String>> testConf =
+        new ConfigBuilder("gravitino.test.stringList")
+            .doc("test")
+            .internal()
+            .stringConf()
+            .toSequence()
+            .create();
+
+    List<String> valueList = testConf.readFrom(configMap);
+    Assertions.assertEquals("test-string1", valueList.get(0));
+    Assertions.assertEquals("test-string2", valueList.get(1));
+    Assertions.assertEquals("test-string3", valueList.get(2));
+
+    valueList.set(0, "test-string4");
+    valueList.set(1, "test-string5");
+    valueList.set(2, "test-string6");
+    testConf.writeTo(configMap, valueList);
+    Assertions.assertEquals(
+        "test-string4,test-string5,test-string6", configMap.get("gravitino.test.stringList"));
+  }
+
+  @Test
+  public void testIntConfValueList() {
+    ConfigEntry<List<Integer>> testConf =
+        new ConfigBuilder("gravitino.test.integerList")
+            .doc("test")
+            .internal()
+            .intConf()
+            .toSequence()
+            .create();
+
+    List<Integer> valueList = testConf.readFrom(configMap);
+    Assertions.assertEquals(1, valueList.get(0));
```

Comment:
Could this method work with `checkValue`?
For example,
```
        new ConfigBuilder("gravitino.test.integerList")
            .doc("test")
            .internal()
            .intConf()
            .checkValue(xxxx)
            .toSequence()
            .create();
```

Reply:
I think it can and it does after my test.
But, `checkValue()`, should only work after function `toSequence()` is invoked as expected.